### PR TITLE
argtable, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
This needs to be merged before a bump for dumb2.0.3

File: dev-libs/argtable/argtable2-13.recipe [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+REVISION="1"
+SOURCE_URI="http://prdownloads.sourceforge.net/argtable/argtable2-13.tar.gz"
+CHECKSUM_SHA256="8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	argtable2$secondaryArchSuffix = $portVersion
+	lib:libargtable2$secondaryArchSuffix = 0.1.8 compat >= 0
```
Comment (alaviss) [inline dev-libs/argtable/argtable2-13.recipe]: Can we get a `libVersion` treatment? :)

File: dev-libs/argtable/argtable2-13.recipe [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+line options with a minimum of fuss. It enables a program's command line \
+syntax to be defined in the source code as an array of argtable structs. \
+The command line is then parsed according to that specification and the \
+resulting values are returned in those same structs where they are accessible \
+to the main program."
+HOMEPAGE="http://argtable.sourceforge.net/"
+COPYRIGHT="1998-2011 Stewart Heitmann"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://prdownloads.sourceforge.net/argtable/argtable2-13.tar.gz"
```
Comment (alaviss) [inline dev-libs/argtable/argtable2-13.recipe]:
```suggestion
SOURCE_URI="https://prdownloads.sourceforge.net/argtable/argtable2-$portVersion.tar.gz"
```

File: dev-libs/argtable/argtable2-13.recipe [hunk 3]
```diff
@@ -0,0 +1,68 @@
+SUMMARY="An ANSI C command line parser"
+DESCRIPTION=" Argtable is an ANSI C library for parsing GNU style command \
+line options with a minimum of fuss. It enables a program's command line \
+syntax to be defined in the source code as an array of argtable structs. \
+The command line is then parsed according to that specification and the \
+resulting values are returned in those same structs where they are accessible \
+to the main program."
+HOMEPAGE="http://argtable.sourceforge.net/"
+COPYRIGHT="1998-2011 Stewart Heitmann"
+LICENSE="GNU GPL v2"
```
Comment (alaviss) [inline dev-libs/argtable/argtable2-13.recipe]:
```suggestion
LICENSE="GNU LGPL v2"
```

File: dev-libs/argtable/argtable2-13.recipe [hunk 4]
```diff
@@ -0,0 +1,68 @@
+SUMMARY="An ANSI C command line parser"
+DESCRIPTION=" Argtable is an ANSI C library for parsing GNU style command \
+line options with a minimum of fuss. It enables a program's command line \
+syntax to be defined in the source code as an array of argtable structs. \
+The command line is then parsed according to that specification and the \
+resulting values are returned in those same structs where they are accessible \
+to the main program."
+HOMEPAGE="http://argtable.sourceforge.net/"
+COPYRIGHT="1998-2011 Stewart Heitmann"
```
Comment (alaviss) [inline dev-libs/argtable/argtable2-13.recipe]:
```suggestion
COPYRIGHT="1998-2001, 2003-2011 Stewart Heitmann"
```

I don't think we should collapse the year like this, given that the author have explicitly put it into two different blocks.

File: dev-libs/argtable/argtable2-13.recipe (hunk 1 above)
Comment (Begasus) [reply dev-libs/argtable/argtable2-13.recipe]: AFAIK this is the last in this line, argtable3 is newer and would mean a rewrite of this I gues ;)

**Pull request merged**